pkg/crypto/hasher.go

```diff
@@ -46,7 +46,7 @@ type Keccak256Hasher struct{}
 func (k *Keccak256Hasher) Hash(data, salt string) string {
 	h := make([]byte, 64)
 	sha3.ShakeSum256(h, []byte(salt+data))
-	return string(h)
+	return base64.StdEncoding.EncodeToString(h)
```

Comment:
need to safely translate the hash to base64 before saving, especially for the policy table with casbin